Runtime/Scripts/InstanceRenderingInfo.cs

```diff
@@ -0,0 +1,238 @@
+﻿using System;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Rendering;
+
+namespace Unity.MeshSync
+{
+    public class InstanceRenderingInfo
```

Comment:
Please make this an internal class, unless we want to open this to users.
If that's the case, we'll also need to write the API documentations.